packages/venia-concept/src/actions/cart/asyncActions.js

```diff
@@ -245,7 +246,7 @@ export const removeItemFromCart = payload => {
             // a price of 0
             const cartItemCount = cart.details ? cart.details.items_count : 0;
             if (cartItemCount === 1) {
-                await clearCartId();
+                lastItemCount = true;
```

Comment:
Since we're not performing the Cart ID reset here anymore, let's move the comment above down to the if block at the bottom.